ecl/hql/hqlexpr.cpp

```diff
@@ -5188,6 +5188,24 @@ CHqlExpression *CHqlExpressionWithType::makeExpression(node_operator _op, ITypeI
     return (CHqlExpression *)e->closeExpr();
 }
 
+CHqlExpression* CHqlExpressionWithType::makeExpression(node_operator op, ITypeInfo *type, const std::initializer_list<IHqlExpression *> &operands)
+{
+    HqlExprArray args;
+    args.ensure(operands.size());
+    for (auto & cur : operands)
+    {
+        //Skip null entries
+        if (cur)
+        {
+#ifdef _DEBUG
+            assertex(QUERYINTERFACE(cur, IHqlExpression));
```

Comment:
Why not dbgassertex (as you used below)

Reply:
I think because the code was copied and modified.  I will change.